.github/workflows/test.yml

```
@@ -24,10 +24,15 @@ jobs:
           - '6_1'
           - '7_0'
           - '7_1'
+          - '7_2'
           - 'latest'
         mysql-version:
           - '5.7'
           - '8.0'
+        exclude:
+          # activerecord-7 doesn't support Ruby 2.6
```

This sentence seems wrong.

```suggestion
          # activerecord-7.2 requires Ruby 3.1.0 or later
```

By the way, "later" might be more appropriate.
cf. https://english.stackexchange.com/questions/112257/above-or-later-when-referencing-a-range-of-versions-of-software

---

Oops, that's my bad. Thanks.